roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 dnsBindAddress: 127.0.0.1:53
 dnsRecursiveResolvConf: /etc/origin/node/resolv.conf
 dnsDomain: {{ openshift.common.dns_domain }}
-dnsIP: {{ openshift_dns_ip }}
+dnsIP: 0.0.0.0
```

Comment:
Should this even be present in the config file, or is 0.0.0.0 mandatory?

Reply:
I don't know if it's required, It's basically a static value. And the configmap is going to override it anyway so I just wanted to minimize change.